keras/preprocessing/image.py

```diff
@@ -746,15 +747,18 @@ def flow(self, x, y=None, batch_size=32, shuffle=True, seed=None,
                 `validation_split` is set in `ImageDataGenerator`.
 
         # Returns
-            An `Iterator` yielding tuples of `(x, y)`
+            An `Iterator` yielding tuples of `(x, y, sample_weight)`
```

Comment:
Docs should be clarified: the output is `(x, y)` unless `sample_weight` is specified.

Reply:
This is clarified at the end of the doc (both the case where `sample_weight` is not specified, as well as the case where both `sample_weight` and `y` are not specified. 

If you want, I can try to rephrase it assuming a default of passing `x` and `y`, but not `sample_weight`.